recipe/meta.yaml

```
@@ -23,14 +23,16 @@ requirements:
     - setuptools
     - setuptools_scm
   run:
-    - python >=3.6
+    - python >=3.7
     - pyyaml >=3.10
     - jmespath >=0.6.2
     - Jsonschema >=3.0.2,<4
     - numpy >=1.10
     - semantic_version >=2.8
     - importlib_resources >=3
     - packaging >=16.0
+    - asdf-standard >=1.0.0
+    - asdf-transform-schemas >=0.2.0 
```

```suggestion
    - asdf-standard >=1.0.1
    - asdf-transform-schemas >=0.2.2 
```
These are the versions that are or will be available on conda-forge starting out.